static/scripts/README

```diff
@@ -5,10 +5,12 @@ This folder contains embedding scripts for Oppia explorations, and tests for
 them. Currently, the tests need to be done manually. They should be run as
 follows:
 
-- Load the 'Welcome' demo exploration (with id '0') on a local development
-  server.
-- Change the content of the initial state to anything you like. This changes
-  the exploration to version 2.
+- Load the 'Welcome' demo exploration (with id '0') for version 0.0.0 and
+  the 'protractor_test_1' exploration (with id '12') for version 0.0.1
+  on a local development server.
+- Change the content of the initial state to anything you like. Change also the
```

Comment:
This is just for the protractor_test_1 exploration, right?

Also, perhaps this couple of paragraphs is a bit obsolete, since we now have the e2e tests. Maybe just delete all the stuff from "TESTING INSTRUCTIONS" all the way to "embedding_tests_dev_0.0.1.html"?